massa-bootstrap/src/server.rs

```diff
@@ -746,7 +729,8 @@ async fn manage_bootstrap<C: NetworkCommandSenderTrait>(
                     server.send_msg(
                         write_timeout,
                         BootstrapServerMessage::BootstrapPeers {
-                            peers: network_command_sender.get_bootstrap_peers().await?,
+                            peers: rt_hack
+                                .block_on(network_command_sender.get_bootstrap_peers())?,
```

Comment:
the idea is more about removing any sort of dependency on a runtime. This is still async because of the requirements of the network command system. here, we are depending on the network command system, not an async runtime (well, we still do, but only via the network command system we already depend on)

changing one runtime for another would defeat the purpose.

Reply:
yeah i know, i suggest that for removing tokio dependency